vmprof/show.py

```diff
@@ -98,9 +98,15 @@ def print_node(parent, node, level):
 
                 elif parts == 1:
                     block_type, funname = node.name.split(':')
-                    p2 = color("JIT code", color.RED, bold=True)
-                    p2b = color('.' * level * self._indent, color.RED, bold=False)
-                    p3 = color(funname, color.WHITE, bold=False)
+                    if "line" == block_type:
```

Comment:
Bikeshedding: this is very uncommon in Python, `block_type == "line"` is more common.